R/fit_SE.R

```diff
@@ -110,3 +76,41 @@ fit_SE <- function(se,
   
   se
 }
+
+
+#' @keywords internal
+fit_FUN <- function(x, 
+                    metric_cols = c(gDRutils::get_header("response_metrics"), 
+                                    "maxlog10Concentration", 
+                                    "N_conc"),
+                    conc = gDRutils::get_env_identifiers("concentration"),
+                    nested_identifiers,
+                    n_point_cutoff,
+                    range_conc,
+                    force_fit,
+                    pcutoff,
+                    cap,
+                    curve_type) {
+  fit_df <- S4Vectors::DataFrame(matrix(NA, 2, length(metric_cols)))
+  colnames(fit_df) <- metric_cols
+  rownames(fit_df) <- c("RV", "GR")
```

Comment:
```suggestion
  rownames(fit_df) <- curve_type
```
@gladkia Please, confirm. (scenario: curve_type = "GR")

Reply:
Since gDRutils::fit_curves uses gDRutils:::.apllyLogisticFit and latter apply fitting in specific manner (firstly for RV, secondly for GR) i will change constructing matrix accordingly.